mobile_wallet/android/mobile_wallet_lib/src/main/java/com/concordium/mobile_wallet_lib/wallet.kt

```diff
@@ -20,8 +20,9 @@ external fun link_check(input: String) : String
 external fun get_identity_keys_and_randomness(input: String) : ReturnValue
 external fun get_account_keys_and_randomness(input: String) : ReturnValue
 external fun sign_transaction(input: String) : ReturnValue
-external fun transaction_to_json(input: String) : ReturnValue
+external fun parameters_to_json(input: String) : ReturnValue
 external fun sign_message(input: String) : ReturnValue
+external fun create_unsigned_transaction(input: String) : ReturnValue
```

Comment:
Is there a reason we should not call this as we do internally, `create_pre_account_transaction`?

Reply:
I was not sure that the preAccountTransaction naming was something we used generally?

Reply:
I changed it.